server/services/claim-service.js

```diff
@@ -4,10 +4,8 @@ const messageService = require('../services/message-service')
 
 module.exports = {
   create: async function (claim) {
-    console.log(claim)

```

Comment:
the console.logs has been removed, I think there is value in logging output when actions take place to assist in monitoring and debugging the application.

For this function useful logging would be writing out whether the claim was existing or new.
Long term we should be switching to a dedicated logging library.

Reply:
Agreed